Sources/Build/Command.link(SwiftModule).swift

```diff
@@ -84,6 +93,11 @@ extension Command {
         case .Library(.Dynamic):
             args.append("-emit-library")
         case .Executable:
+            // FIXME: This is inefficient, we can store the main module in enum payload.
```

Comment:
I'd prefer that the reasoning be given in the comment, even if it has to be a block comment.  External references are sometimes unavoidable but shouldn't be a substitute for explaining the semantics of what's going on.